truffle/src/main/ruby/core/byte_array.rb

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2016 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 1.0
+# GNU General Public License version 2
+# GNU Lesser General Public License version 2.1
+# OTHER DEALINGS IN THE SOFTWARE.
+
+module Rubinius
+class ByteArray
+
```

Comment:
None of the nested Rubinius module code is indented twice. I just followed that style. E.g., see the various `Rubinius::FFI` classes.